docs/content/concepts.mdx

```diff
@@ -1,89 +1,241 @@
+---
```

Comment:
The goal of this page is to act as a jumping-off point, so the copy should be kept high-level and brief.